### PR TITLE
Wip/superm1/flatpak fixes

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

File: src/fu-tool.c [hunk 1]
```diff
@@ -63,6 +63,15 @@ fu_sort_command_name_cb (FuUtilItem **item1, FuUtilItem **item2)
 	return g_strcmp0 ((*item1)->name, (*item2)->name);
 }
 
+static void
+fu_util_test_for_sandbox (const gchar *value)
```
Comment (hughsie) [inline src/fu-tool.c]: couldn't this just take the GError as a parameter and use `g_prefix_error()`? It seems a shame to just put it in a `g_warning`... Ack on the first two patches tho.

Comment (superm1) [reply src/fu-tool.c]: fixed

File: src/fu-tool.c [hunk 2]
```diff
@@ -63,6 +63,16 @@ fu_sort_command_name_cb (FuUtilItem **item1, FuUtilItem **item2)
 	return g_strcmp0 ((*item1)->name, (*item2)->name);
 }
 
+static void
+fu_util_test_for_sandbox (const gchar *value, GError **error)
+{
+	g_autofree gchar *path = g_path_get_dirname (value);
+	if (!g_file_test (path, G_FILE_TEST_EXISTS | G_FILE_TEST_IS_DIR))
```
Comment (hughsie) [inline src/fu-tool.c]: you probably want braces here -- it's a bit lopsided. it' also needs to be called something like fu_util_perhaps_prefix_sandbox_error() as it's not actually testing for a sandbox, just making an error better -- better names welcome.

Comment (superm1) [reply src/fu-tool.c]: fixed

**Comment by hughsie** [comment]
Looks good to me, but CI looks like it's failing... unrelated?

**Comment by superm1** [comment]
Yeah Travis is having some availability problems today on their sudo jobs according to their page.

**Pull request merged**